Components/toast/toast.css

```diff
@@ -0,0 +1,39 @@
+
+
+.example-container .display-toast {
+    position: relative;
+    top: 0;
+    right: 0;
+    display: block;
+}
+.toast {
+    transition: ease-in 300ms;
+    width: fit-content;
+    padding: 1rem 1rem 1rem 1rem;
+    position: fixed;
+    margin-left: auto;
+    margin-right: auto;
+    right: 3rem;
+    bottom: 3rem;
+    background-color: white;
+    color: var(--secondary-text-color);
+    border-radius: 0.6rem;
+    display: none;
+    z-index: 5;   
+} 
+
+
+.toast-success {
+    background-color: #84f1c9;
```

Comment:
Use global variables for colors to have consistency throughout the code.

Reply:
Hey, Kavita thanks for putting your time into reviewing this project!
Thanks for the appreciation! and like I have told Anjali above, I have used all these 6 colors distinctively in toast CSS only. and it's getting too heavy to manage colors in reset CSS.